bash-cli.inc.sh

```diff
@@ -230,9 +230,16 @@ function bcli_bash_completions() {
                 return
             fi
         done
-
-        # shellcheck disable=SC2207 # Using this as alternatives are not cross-platform or introduce dependencies
-        COMPREPLY=($(compgen -W '--help' -- "$curr_arg"))
+        # Use bash completion file if any.
+        if [ -f "${cmd_file}.complete" ]; then
+            # shellcheck disable=SC2207 # Using this as alternatives are not cross-platform or introduce dependencies
```

Comment:
I think this might work to suppress the error: https://github.com/koalaman/shellcheck/wiki/SC1090#exceptions

```suggestion
            # shellcheck disable=SC2207 source=/dev/null # Using this as alternatives are not cross-platform or introduce dependencies
```